src/test/java/dumshenko/daniil/ShapeTest.java

```diff
@@ -0,0 +1,28 @@
+package dumshenko.daniil;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ShapeTest {
+
+    @Test
+    public void testCircleArea() {
```

Comment:
nit: Let's follow the `should...` test naming convention 😉 
```suggestion
    public void shouldCorrectlyComputeCircleArea() {
```